platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/IdpHandler.java

```diff
@@ -606,12 +604,23 @@ private String encodeRedirectRequest(String request) throws WSSecurityException,
     return URLEncoder.encode(RestSecurity.deflateAndBase64Encode(request), "UTF-8");
   }
 
-  private String encodePostRequest(String request) throws WSSecurityException, IOException {
+  private String encodePostRequest(String request) throws WSSecurityException {
     return Base64.getEncoder().encodeToString(request.getBytes(StandardCharsets.UTF_8));
   }
 
   private String recreateFullRequestUrl(HttpServletRequest request) {
     StringBuffer requestURL = request.getRequestURL();
+
+    try {
```

Comment:
✏️ It's not the biggest win in the world, but this block of code and the block in `LogoutRequestService` are almost identical (one gets a new `String` and the other creates a new `StringBuffer` around it) and could be slightly refactored.